MyWeight/Extensions/UITableViewCell+ReuseIdentifier.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  UITableViewCell+ReuseIdentifier.swift
+//  MyWeight
+//
+//  Created by Bruno Koga on 22/06/16.
+//  Copyright © 2016 Diogo Tridapalli. All rights reserved.
+//
+
+import UIKit
+
+extension UITableViewCell {
+    /**
+     - returns: a UINib with the default Nib name (matches the class name).
+        If the nib file is not found, this will cause a exception crash.
+     */
+    static func nib() -> UINib {
+        return UINib(nibName: defaultNibName, bundle: nil)
+    }
+    
+    /**
+     Default nib name for this class (matches the class name).
+     */
+    static var defaultNibName: String {
+        return String(self)
+    }
+    
+    /**
+     Default reuse identifier for this class (matches the class name).
+     */
+    static var defaultReuseIdentifier: String {
+        return String(self)
+    }
+}
+
+extension UITableView {
+    
+    /**
+     Register a cell into the tableView.
+     Uses the defaultReuseIdentifier if the reuseIdentifier is not provided.
+     
+     - parameter cellClass: the class to register.
+     */
+    func registerCellNib(cellClass: UITableViewCell.Type, forCellReuseIdentifier reuseIdentifier: String? = nil) {
```

Comment:
We'll, funny thing is on Playground it seems to work. 

If I change the method signature, I see this error, meaning that de `dequeueReusableCellWithIdentifier` returned nil. My question is: why doesn't it return nil when I used the old signature?

![screen shot 2016-06-24 at 16 48 00](https://cloud.githubusercontent.com/assets/1219990/16341062/8da7140a-3a2b-11e6-95c0-510c73a938ec.png)